docs/content/concepts/apis/built-in.md

```diff
@@ -1,8 +1,14 @@
-# Built-in APIs
+# Kubernetes API Endpoints used by kcp
 
-kcp includes some, but not all, of the APIs you are likely familiar with from Kubernetes:
+kcp includes some, but not all, of the Kubernetes APIs.
 
-## (core) v1
+kcp does not make us of Kubernetes API endpoints that are concerned with orchestrating clusters and their workloads. Clusters dedicated to workload orchestration have these resources but kcp does not.
```

Comment:
this sounds bit of strange too. 
I think we need to be more explicit:

```
kcp functions as a pure control plane, providing only a subset of the full Kubernetes API. It is not a complete Kubernetes distribution and intentionally excludes APIs related to running workloads.

Because kcp doesn't manage nodes or run containers, it lacks the necessary controllers and resource definitions for orchestration. Specifically, you will not find key resource types such as:

The apps API group (e.g., Deployments, StatefulSets, DaemonSets)
The core Pods resource
Networking resources like Services and Endpoints
You can find more details about the specific APIs kcp does support below.
```